pkg/sql/codegen/pai/codegen.go

```diff
@@ -68,41 +45,6 @@ func FormatCkptDir(modelName string) (string, error) {
 	return strings.Join([]string{ossDir + "/", ossURIParts[1]}, "?"), nil
 }
```

Comment:
```go
func CheckpointURL(modelName string) (string, error) {
    dd := strings.Split(os.Getenv("SQLFLOW_OSS_CHECKPOINT_DIR"), "?")
    if len(dd) != 2 {
        return "", fmt.Errorf("Must set SQLFLOW_OSS_CHECKPOINT_DIR to oss://bucket/...?role_arn=...")
    } 
    return strings.Join([]string{path.Join(dd[0], modelName), dd[1]}, "?"), nil
}
```

Reply:
Done. Not using `path.Join` because oss URIs must use `/` and it's not os specific.